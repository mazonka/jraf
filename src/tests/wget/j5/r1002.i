command=jw md 0 / : save 0 /a 6 WGE-IGFh : ping