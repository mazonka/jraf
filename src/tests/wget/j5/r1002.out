OK  : OK 6 : OK